app/_hub/kong-inc/jwt/index.md

```diff
@@ -102,42 +107,45 @@ params:
+        An optional string (consumer uuid) value to use as an anonymous consumer if authentication fails. If empty (default), the request will fail with an authentication failure `4xx`. The anonymous value must refer to the Consumer `id` attribute that is internal to Kong, and **not** its `custom_id`.
     - name: run_on_preflight
       required: false
       default: "`true`"
       description: |
-        A boolean value that indicates whether the plugin should run (and try to authenticate) on `OPTIONS` preflight requests, if set to `false` then `OPTIONS` requests will always be allowed.
+        A boolean value that indicates whether the plugin should run (and try to authenticate) on `OPTIONS` preflight requests. If set to `false`, then `OPTIONS` requests will always be allowed.
     - name: maximum_expiration
       required: false
       default: 0
       description: |
-        An integer limiting the lifetime of the JWT to `maximum_expiration` seconds in the future. Any JWT that has a longer lifetime will rejected (HTTP 403). If this value is specified, `exp` must be specified as well in the `claims_to_verify` property. The default value of `0` represents an indefinite period. Potential clock skew should be considered when configuring this setting.
+        An integer limiting the lifetime of the JWT to `maximum_expiration` seconds in the future. Any JWT that has a longer lifetime is rejected (HTTP 403). If this value is specified, `exp` must be specified as well in the `claims_to_verify` property. The default value of `0` represents an indefinite period. Potential clock skew should be considered when configuring this setting.
 
   extra: |
     <div class="alert alert-warning">
-        <center>The option `config.run_on_preflight` is only available from version `0.11.1` and later</center>
+        <center>The option `config.run_on_preflight` is only available from version `0.11.1` and later.</center>
     </div>
 
 ---
 
 ## Documentation
 
-In order to use the plugin, you first need to create a Consumer and associate one or more JWT credentials (holding the public and private keys used to verify the token) to it. The Consumer represents a developer using the final service.
+To use the plugin, you first need to create a Consumer and associate one or more
+JWT credentials (holding the public and private keys used to verify the token) to it.
+The Consumer represents a developer using the final service.
 
 ### Create a Consumer
 
 You need to associate a credential to an existing [Consumer][consumer-object] object.
 A Consumer can have many credentials.
 
-{% tabs %}
-{% tab With a Database %}
+{% navtabs %}
+{% navtab With a Database %}
```

Comment:
```suggestion
{% navtab Kong Admin API %}
```
Let's rename these tabs to match the renamed examples, as this isn't the true dichotomy - you could use a declarative config file with a database too, there's nothing stopping that.